config/swagger.yaml

```diff
@@ -218,6 +218,31 @@ paths:
           schema:
             $ref: '#/definitions/Error'
       security: []
+  /name:
+    get:
+      tags:
+        - external
+      operationId: GetName
+      description: 'Get name entry from Naming System'
+      consumes:
+        - application/json
+      produces:
+        - application/json
+      parameters:
+        - in: query
+          name: name
+          description: 'Name to get entry from Naming System'
+          type : string
+      responses:
+        '200':
+          description: Name data
+          schema:
+            $ref: '#/definitions/NameEntry'
+        '404':
+          description: Name not found
+          schema:
+            $ref: '#/definitions/Error'
+      security: []
```

Comment:
Since [this commit](https://github.com/aeternity/epoch/commit/7d45685f9e1faab407ec01117749c9824406c2eb) of @ThomasArts , we should remove the `security` clause